executor/aggregate.go

```diff
@@ -840,6 +844,11 @@ func (e *HashAggExec) parallelExec(ctx context.Context, chk *chunk.Chunk) error
 
 // unparallelExec executes hash aggregation algorithm in single thread.
 func (e *HashAggExec) unparallelExec(ctx context.Context, chk *chunk.Chunk) error {
```

Comment:
What about the parallelExec logic?

Reply:
In this pr, we just consider the unparallelExec logic.